abipy/examples/plot/plot_lumi_1D_Eu_doped.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+r"""
+Luminescent properties (1D-CCM)
+====================
+
+This example shows how to post-process the results of a LumiWork
+following a one-dimensional configuration coordinate model (1D-CCM).
+Based on a Eu-doped phosphor SrAlLi3N4:Eu presenting two non-equivalent sites for Eu
+See example/flows/run_lumi_Eu_doped_SLA.py
+"""
+
+#%%
+# Read the 4 points netcdf file produced by the two LumiWork
+
+from abipy.lumi.deltaSCF import DeltaSCF
+import abipy.data as abidata
+import pandas as pd
+
+SLA_site_1 = DeltaSCF.from_four_points_file([abidata.ref_file("site_1_relaxed_gs_out_GSR.nc"),
```

Comment:
It would be useful to reuse this piece of code to write a set of unit tests for DeltaSCF in lumi/tests/test_deltascf.py
I usually add tests to make sure that the plotting tools continue to work with more recent versions of matplotlib. See https://github.com/abinit/abipy/blob/7229e1e34782c06167a1a4975f736fe82d6e48af/abipy/electrons/tests/test_ebands.py#L184 for inspiration.